test/src/main/java/org/apache/accumulo/test/NewTableConfigurationIT.java

```diff
@@ -534,14 +534,12 @@ public void testConflictsWithDefaults() throws Exception {
        */
       // add an iterator with same priority as the default iterator
       var iterator1 = new IteratorSetting(20, "foo", "foo.bar");
-      var exception = assertThrows(IllegalStateException.class, () -> client.tableOperations()
-          .create(table, new NewTableConfiguration().attachIterator(iterator1)));
-      assertTrue(exception.getMessage().contains("iterator priority conflict"));
+      assertThrows(IllegalStateException.class, () -> client.tableOperations().create(table,
+          new NewTableConfiguration().attachIterator(iterator1)));
```

Comment:
What happened with the exception messages?

Reply:
Posted this comment same time as your question: https://github.com/apache/accumulo/pull/6037#issuecomment-3687202078